### PR TITLE
Add a note for whitelisted tags by default in the TargetScrubber

### PR DESCRIPTION
**Pull request by paul-mesnilgrente**
See https://github.com/rails/rails-html-sanitizer/issues/109.

What do you think about the wording?

### CODE REVIEW

**Comment by s01ipsist** [comment]
IMHO, a deep link to https://github.com/flavorjones/loofah/blob/main/lib/loofah/html5/safelist.rb would be valuable

File: README.md [hunk 1]
```diff
@@ -83,6 +83,8 @@ html_fragment.to_s # => "<a></a>"
 Where `PermitScrubber` picks out tags and attributes to permit in sanitization,
 `Rails::Html::TargetScrubber` targets them for removal.
 
+**Note:** by default, it will scrub anything that is not part of the whitelisted tags from
```
Comment (kaspth) [inline README.md:86]:
```suggestion
**Note:** by default, it will scrub anything that is not part of the permitted tags from
```

**Review by kaspth — COMMENTED** [review]
Thanks! Needs a wording change, and the suggested deep link would also be good.

**Comment by paul-mesnilgrente** [comment]
What do you think  now?

**Comment by kaspth** [comment]
Thanks!

**Pull request merged**